keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -800,6 +809,8 @@ feature enablement causes the number of syncs to increase.
 
 #### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
+No.
```

Comment:
How about a metric for created pods, maybe with a reason: (new/replacement)

Reply:
I updated the KEP to say that we are adding a new metric to help with this.  

The additional metric is part of our beta deliverables.